### PR TITLE
New stuff

### PR DESCRIPTION
**Pull request by Koriath**
3 Diferent ways to get the Spirit Guardians
Minor fix to Inert Virus
Added "Purge" virus to counteract liggertittis. Done with FEV.
Added "Explosive" virus. Done with FEV.

### CODE REVIEW

**Comment by Koriath** [comment]
My .dme file seems borked or too diferent.
You'll need to activate the following files:
"explosive.dm" , "purge.dm" and "scarab.dm", all in th symptoms folder.

File: code/modules/research/designs/weapon_designs.dm [hunk 1]
```diff
@@ -22,6 +22,14 @@
 	build_path = /obj/item/device/firing_pin/implant/loyalty
 	category = list("Firing Pins")
 
+/datum/design/nanomachine_injector
+	name = "Parasitic Nanomachine Injector"
+	desc = "Though powerful in combat, these nanomachines require a living host as a source of fuel and home base."
+	id = "parasitic_nano_injector"
+	req_tech = list("combat" = 5, "materials" = 4, "bluespace" = 4)
```
Comment (octareenroon91) [inline code/modules/research/designs/weapon_designs.dm]: Put a 6 or 7 in here please

File: code/modules/reagents/Chemistry-Recipes/Others.dm [hunk 2]
```diff
@@ -124,6 +126,40 @@
 			D.RemoveSymptom(healer)
 			D.AddSymptom(new /datum/symptom/regen())
 			D.Refresh(1) //It needs to actually change the disease name for reasons
+		//scarab symptom
+		var/itching = locate(/datum/symptom/itching) in D.symptoms
+		var/hallucinogen = locate(/datum/symptom/hallucigen) in D.symptoms
+		var/vomit = locate(/datum/symptomvomit/blood) in D.symptoms
```
Comment (octareenroon91) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: Fix the typepath
code/modules/reagents/Chemistry-Recipes/Others.dm:132:error: /datum/symptomvomit/blood: undefined type path

File: code/modules/reagents/Chemistry-Recipes/Others.dm [hunk 3]
```diff
@@ -124,6 +126,40 @@
...
+		var/liggeritis = locate(/datum/symptom/liggeritis) in D.symptoms
+		var/metabolism = locate(/datum/symptom/heal/metabolism) in D.symptoms
+		if(liggeritis && metabolism)
+			D.RemoveSymptom(liggeritis)
+			D.RemoveSymptom(metabolism)
+			D.AddSymptom(new /datum/symptom/purge())
+			D.Refresh(1)
+		//sensory restoration
+		var/visionaid = locate(/datum/symptom/visionaid) in D.symptoms
+		var/youth = locate(/datum/symptom/heal/youth) in D.symptoms
```
Comment (octareenroon91) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: Please double check this typepath.

code/modules/reagents/Chemistry-Recipes/Others.dm:157:error: /datum/symptom/heal/youth: undefined type path

**Comment by Koriath** [comment]
Everything fixed, should be good to go.

**Comment by Koriath** [comment]
Just a warning, people have been requesting this one because of the spawn guardians. IF there's anything missing, say so and I'll add it. Otherwise, please merge ASAP.

**Comment by octareenroon91** [comment]
K

**Pull request merged**